sycl/include/CL/sycl/detail/helpers.hpp

```diff
@@ -40,7 +40,8 @@ getOrWaitEvents(std::vector<cl::sycl::event> DepEvents,
 
 void waitEvents(std::vector<cl::sycl::event> DepEvents);
 
-struct Builder {
+class Builder {
```

Comment:
`Builder` struct is not part of the SYCL specification.
Did you decide to avoid using` struct` for all types in our project - even for internal?

Reply:
That is already 3rd time when I fix something for Builder. Have no idea why, but Builder tends to be treated as class by other developers.